utils/trackerbot.py

```diff
@@ -197,11 +197,14 @@ def templates_to_test(api, limit=1):
 
     """
     templates = []
-    for pt in api.untestedtemplate.get(limit=limit, tested=False).get('objects', []):
+    for pt in api.untestedtemplate.get(
+            limit=limit, tested=False, provider__type=request_type).get(
```

Comment:
Yes, it works. 
tried below cases:
get(api)
get(api, 'openstack')
get(api, 'virtualcenter')
get(api, 'rhevm')